### PR TITLE
Add Travis support

### PR DESCRIPTION
**Pull request by traversaro**
Add script for running automatic tests on Travis. 

Depends on https://github.com/robotology/gazebo-fmi/pull/23 being merged.

### CODE REVIEW

File: .ci/install_debian_and_script.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# CI specific packages
+apt-get install -y clang git
+
+# Dependencies
+apt-get install -y libgazebo9-dev openmodelica
+
+# Script
+cd $TRAVIS_BUILD_DIR
+mkdir build && cd build
+cmake -DBUILD_TESTING:BOOL=ON -DUSES_SYSTEM_FMILIBRARY:BOOL=OFF -G"${TRAVIS_CMAKE_GENERATOR}" ..
```
Comment (prashanthr05) [inline .ci/install_debian_and_script.sh]: -DUSE_SYSTEM_FMILIBRARY:BOOL=OFF

Comment (traversaro) [reply .ci/install_debian_and_script.sh]: Fixed in  https://github.com/robotology/gazebo-fmi/pull/24/commits/bfa4d0586a86fb4648727f8fad44c23b6c606c00 .

**Comment by prashanthr05** [comment]
Probably also pull the `master` onto this branch to reflect the changes after the `cmake/FetchContent` merge ?

**Comment by traversaro** [comment]
> Probably also pull the master onto this branch to reflect the changes after the cmake/FetchContent merge ?

This branch was already based on `cmake/FetchContent`, everything should work fine, but now we can wait for Travis to let us know. :D

**Comment by traversaro** [comment]
Travis is successful, merging. .

**Pull request merged**